source/include/jobs.h

```diff
@@ -192,6 +292,13 @@ typedef enum
  *
  * @note The thingName parameter does not need a NUL terminator.
  *
+ * @note The AWS IoT Jobs service does not require clients to subscribe
+ * to the "/accepted" and "/rejected" response topics for the APIs that
+ * accept PUBLISH requests. The Jobs service will send responses to requests
```

Comment:
Could change wording here, "publish requests" sounds like a request for a publish instead of a request delivered via publish
```suggestion
 * accept published requests. The Jobs service will send responses to requests
```

Reply:
I have reworded to address the readability concern.